core/exceptions.py

```diff
@@ -1,50 +1,16 @@
-class BaseError(Exception):
-    """Base class for all errors"""
+class BasePalaceException(Exception):
+    """Base class for all Exceptions in the Palace manager."""
 
-    def __init__(
-        self, message: str | None = None, inner_exception: Exception | None = None
-    ):
+    def __init__(self, message: str | None = None):
         """Initializes a new instance of BaseError class
 
         :param message: String containing description of the error occurred
-        :param inner_exception: (Optional) Inner exception
```

Comment:
error -> exception that 

And the inner_exception has gone away yes?

Reply:
Yes, `inner_exception` was removed. Its leftover from Python 2 days when there was no built in way to chain exceptions. Now that we are on Python 3 I replaced it with PEP3134 exception chaining in the couple places where `inner_exception` was previously used.